nmdc_server/ingest/study.py

```diff
@@ -41,9 +42,13 @@ def get_or_create_pi(db: Session, name: str, url: Optional[str], orcid: Optional
 class Study(StudyCreate):
     _extract_value = validator("*", pre=True, allow_reuse=True)(extract_value)
 
-    @root_validator(pre=True)
+    @model_validator(mode="before")
     def extract_extras(cls, values):
-        return extract_extras(cls, values)
+        return extract_extras(cls, values)  # type: ignore
+
+
+List = list
```

Comment:
Out of curiosity, does adding `from typing import List` to the top of the file remove the need for these 2 lines?

Reply:
Yes, it does

Reply:
See 4e2f5af